### ISSUE TITLE
Tests (general and integration) scripts are platform dependent and won't run in Windows

### ISSUE THREAD

**Issue by xxMattewxx** [ProjectOpenSea/opensea-js#1135]
### Component

Other (please describe)

### Have you ensured that all of these are up to date?

- [X] opensea-js
- [X] Node (minimum v16)

### What version of opensea-js are you on?

6.1.5 (built from main branch)

### Operating System

Windows

### Describe the bug

NPM scripts for tests work fine in Linux (Ubuntu 22.04.02) but fail in Windows 10 (build 19044) due to the way environment variables are set

With those commands:
```json
{
    "test": "TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' TS_NODE_TRANSPILE_ONLY=true nyc mocha -r ts-node/register test/**/*.spec.ts --exclude test/integration/**/*.ts --timeout 15000",
    "test:integration": "TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' TS_NODE_TRANSPILE_ONLY=true nyc mocha -r ts-node/register -r dotenv/config test/integration/**/*.spec.ts --timeout 25000"
}
```
I receive error:
`'TS_NODE_COMPILER_OPTIONS' is not recognized as an internal or external command,
operable program or batch file.`

I fixed this by installing https://github.com/kentcdodds/cross-env and switching to:
```json
{
    "test": "cross-env TS_NODE_COMPILER_OPTIONS={\\\"module\\\":\\\"commonjs\\\"} TS_NODE_TRANSPILE_ONLY=true nyc mocha -r ts-node/register test/**/*.spec.ts --exclude test/integration/**/*.ts --timeout 15000",
    "test:integration": "cross-env TS_NODE_COMPILER_OPTIONS={\\\"module\\\":\\\"commonjs\\\"} TS_NODE_TRANSPILE_ONLY=true nyc mocha -r ts-node/register -r dotenv/config test/integration/**/*.spec.ts --timeout 25000"
}
```

**Comment by ryanio** [comment]
Thanks for reporting. Would you like to open a PR with your fix? I can also do.

**Comment by mchmatt** [comment]
PR sent (was logged in as @xxMattewxx before... oops): https://github.com/ProjectOpenSea/opensea-js/pull/1136

**Comment by ryanio** [comment]
have released as [v6.1.7](https://github.com/ProjectOpenSea/opensea-js/releases/tag/v6.1.7)!
